modules.d/35network-legacy/ifup.sh

```diff
@@ -18,11 +18,162 @@ netif=$1
 
 # loopback is always handled the same way
 if [ "$netif" = "lo" ] ; then
+    # systemd probably has already set up lo
+    ip link show dev lo >/dev/null && exit 0
+
     ip link set lo up
     ip addr add 127.0.0.1/8 dev lo
     exit 0
 fi
 
+dhcp_backend() {
+    type wicked >/dev/null 2>&1 && \
+    echo "wicked" || \
+    echo "dhclient"
```

Comment:
I'd rather this either be flipped or set up via configuration. This would be an unexpected nasty surprise for any openSUSE user who is trying to avoid wicked (such as myself).

Reply:
yes please